file-formats/dobj/README.md

```diff
@@ -0,0 +1,5 @@
+# DOBJ File Format
+
+File | Cardinality | Definition | Schema | Example
+:--- | :---  | :--- | :--- | :---
+`<name>.dobj.json` | 1 | [`zif_aff_dobj_v1.intf.abap`](./type/zif_aff_dobj_v1.intf.abap) | [`dobj-v1.json`](./dobj-v1.json) | [`z_aff_example_dobj.dobj.json`](./examples/z_aff_example_dobj.dobj.json)
```

Comment:
Either an example object could be provided, or we create an issue and remove the link to solve also the broken link check issue